### PR TITLE
[homepage] [s] Small changes on new homepage

### PR DESCRIPTION
**Pull request by nathenf**
It took me a lot of time to get set up using VS Code etc and haven't made a huge impact. 

- Lists 3 key articles
- Lists our top 3 initiatives

### CODE REVIEW

**Comment by rufuspollock** [comment]
👏 @nathenf  it is working https://6a481c30.lifeitself-org.pages.dev/home-new - well done.

File: content/residencies-and-courses/index
Comment (rufuspollock) [inline content/residencies-and-courses/index:1]: @nathenf i think you add this page by mistake as we have programs ...

**Review by rufuspollock — CHANGES_REQUESTED** [review]
All looks good except for accidentally added page. I'd suggest closing this PR and just pushing directly to main with the pages to home-new (no need for a PR again).

File: content/residencies-and-courses/index
Comment (rufuspollock) [reply content/residencies-and-courses/index:1]: Or we can just merge ... and you remove later 😆

Comment (nathenf) [reply content/residencies-and-courses/index:1]: I'll merge and remove. I'm not sure how this happened...

**Pull request merged**